userguide/src/main/jbake/content/attributes.conf

```diff
@@ -1,27 +1,33 @@
-:TechnologyFullName: Debugging Support for Other Languages
+:TechnologyFullName: Jakarta Debugging Support for Other Languages
 :TechnologyShortName: Debugging Support for Other Languages
-:TechnologyVersion: 1.0
-:TechnologyRI: Eclipse Glassfish 5.0
+:LegacyAcronym: DSOL
+:TechnologyVersion: 2.0
+:TechnologyRI: Eclipse Glassfish 6.0
 :TechnologyRIURL: http://javaee.github.io/glassfish
-:JSR: 045
-:JSRURL: http://jcp.org/en/jsr/detail?id=045
-:techID: OL
-////
-The environment variable used to specify the home directory for the technology.
-Used in config.inc.
-////
-:TechnologyHomeEnv: WOMBAT_HOME
-////
-Java SE version required.
-////
+:SpecificationURL: https://jakarta.ee/specifications/debugging/2.0/
+:TCKInquiryList: mailto:jakartaee-tck-dev@eclipse.org[jakartaee-tck-dev@eclipse.org]
+:SpecificationInquiryList: mailto:jsp-dev@eclipse.org[jsp-dev@eclipse.org]
+:techID: DSOL
+// Define this attribute (uncomment it) if the TCK includes no API tests.  (Rare.)
+:no-api-tests:
+// Define this attribute (uncomment it) if the TCK includes end-to-end tests.
+// :end-to-end-tests:
+// Define this attribute (uncomment it) if subsets of the API are allowed.
+// (Common Annotations only)
+// :subset-allowed:
+//
+// The environment variable used to specify the home directory
+// for the technology.  Used in config.inc.
+:TechnologyHomeEnv: DSOL_HOME
+// Java SE version required.
 :SEversion: 8
 :JavaTestVersion: T.T.T
 :jteFileName: <TS_HOME>/bin/wombat.jte
 :jtxFileName: <TS_HOME>/bin/wombat.jtx
-:TCKPackageName: wombat-x.x_dd-Mmm-YYYY.zip
-////
-Directory names used in examples in using.adoc.
-////
+:TCKPackageName: debugging-tck-2.0.0.zip
```

Comment:
I think the tck name expected here is the eftl versioned as "jakarta-debugging-tck-2.0.0.zip"